presto-docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,37 @@
+===================
+Snowflake Connector
+===================
+
+The Snowflake connector allows querying and creating tables in an
+external Snowflake account. This can be used to join data between
+different systems like Snowflake and Hive, or between two different
+Snowflake accounts.
+
+Configuration
+-------------
+
+To configure the Snowflake connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``snowflake.properties``, to
+mount the Redshift connector as the ``snowflake`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=snowflake
+    connection-url=jdbc:snowflake://<account>.snowflakecomputing.com>/?db=<database>
```

Comment:
Remove extra > in `snowflakecomputing.com>`

Reply:
👍